src/features/identityCheck/pages/helpers/mapStepsDetails.ts

```diff
@@ -0,0 +1,33 @@
+import { SubscriptionStepCompletionState, SubscriptionStepperResponse } from 'api/gen'
+import { StepButtonState, StepConfigNewStepper, StepDetails } from 'features/identityCheck/types'
+
+export const mapStepsDetails = (
+  stepToCompleteList: SubscriptionStepperResponse['subscriptionStepsToDisplay'],
```

Comment:
```suggestion
  stepsToDisplay: SubscriptionStepperResponse['subscriptionStepsToDisplay'],
```
Les étapes affichées ne seront pas forcément à compléter